src/signals/incident/components/form/MapSelectors/Asset/Selector/LegendPanel/LegendPanel.tsx

```diff
@@ -22,6 +22,7 @@ const LegendPanel: FunctionComponent<LegendPanelProps> = ({
   <Panel
     className={`${className} ${slide}`}
     data-testid="legendPanel"
+    id="legendPanel"
```

Comment:
is it related to the aria attributes that we need the id's on legendPanel and addressPanel?